usaspending_api/disaster/tests/integration/test_disaster_agency_spending.py

```diff
@@ -87,27 +87,16 @@ def test_basic_success(client, disaster_account_data, elasticsearch_award_index,
             "code": "008",
             "description": "Agency 008",
             "children": [],
-            "award_count": 2,
-            "obligation": 22000.0,
-            "outlay": 20000.0,
-            "total_budgetary_resources": None,
-        },
-        {
-            "id": 1,
-            "code": "007",
-            "description": "Agency 007",
-            "children": [],
             "award_count": 1,
-            "obligation": 222.0,
-            "outlay": 0.0,
+            "obligation": 2000.0,
+            "outlay": 20000.0,
```

Comment:
The issue seems to be that when loading the data from postgres to ES, the postgres function `now()` is not mocking the data with the monkeypatch correctly, so it isn't loading all the data in the disaster_account_data since some of it is technically in the future. If we change the test data to be in the past, the tests behave correctly.

Reply:
Much closer to original values now (with award count going from 1 to 2 making 7)
https://github.com/fedspendingtransparency/usaspending-api/blob/9d8f021938431591c8b4b1b13e423706790966dc/usaspending_api/disaster/tests/integration/test_disaster_agency_spending.py#L115